recipes/simbody/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.scm import Version
+from conan.tools.files import copy, get, rmdir, apply_conandata_patches
+import os
+
+required_conan_version = ">=1.50.0"
+
+class SimbodyConan(ConanFile):
+    name = "simbody"
+    description = "High-performance, open-source toolkit for science- and engineering-quality simulation"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/simbody/simbody"
+    topics = ("high-performance", "science", "simulation")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    def validate(self):
+        if not self.dependencies["openblas"].options.build_lapack:
+            raise ConanInvalidConfiguration(f"{self.name} requires the openblas:build_lapack: True")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("openblas/0.3.17")
+        self.requires("opengl/system")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version],  
+                    destination=self.source_folder, strip_root=True)
+
+    def layout(self):
+        cmake_layout(self, src_folder="Simbody/src")
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, "LICENSE", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        apply_conandata_patches(self)
+        cmake = cmake = CMake(self)
+        cmake.install()
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def package_info(self):
+        version_major = Version(self.version).major
+        simbody_cmake_component = f"simbody{version_major}"
```

Comment:
Maybe I am looking at the worng code but I did not find this.

it seems like the cmake tragets have these names https://github.com/simbody/simbody/blob/a8f49c84e98ccf3b7e6f05db55a29520e5f9c176/Simbody/CMakeLists.txt#L45